scripts/usb_scan.py

```diff
@@ -374,10 +375,15 @@ def check_hex_length(self, name, value):
         return False
 
     @staticmethod
-    def log_parse_error(pos, end, target, line):
-        log.error(
-            "Malformed policy rule, unable to parse '{}'. Ignored line: {}"
-            .format(target[pos:end], line))
+    def log_exit(m):
+        log.error(m)
+        sys.exit(m)
+
+    @staticmethod
+    def parse_error(pos, end, target, line):
+        Policy.log_exit(
```

Comment:
Why not `self.log_exit`?

Reply:
OK. move log_exit() to main scope.

Because it's a staticmethod,
after moving it to main scope,  I can call it.